### PR TITLE
Tech arch principles feedback

### PR DESCRIPTION
**Pull request by saliceti**

### CODE REVIEW

**Comment by andyjpb** [comment]
Looks good to me!

File: source/principles/architecture/index.html.md.erb [hunk 1]
```diff
@@ -34,10 +34,11 @@ Design for user needs identified by research, not for generic reuse - that is pr
...
 
-## Stand on the shoulders of giants
-Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic  and make use of existing technologies where it facilitates your specific needs.  Focusing on what makes your service different is a strategic advantage, and using existing services.
+## Focus on creating value
+Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic and make use of existing technologies where it facilitates your specific needs. Focusing on what makes your service different is a strategic advantage, and using existing services.
```
Comment (misaka) [inline source/principles/architecture/index.html.md.erb:38]: Slight adjustment to remove repetition and to make a contrasting compound sentence clearer.

```suggestion
Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic advantages. Make use of existing technologies and services where it facilitates your specific needs.
```

File: source/principles/architecture/index.html.md.erb [hunk 2]
```diff
@@ -34,10 +34,11 @@ Design for user needs identified by research, not for generic reuse - that is pr
...
+
+Do away with the “undifferentiated heavy lifting” of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor-lock in and unnecessary spending by researching and using open source software when appropriate.
```
Comment (misaka) [inline source/principles/architecture/index.html.md.erb:40]: Feels like the double-quotes here makes us unsure of the name here, or at least I think we should use a slightly more assertive tone here. 

```suggestion
Do away with the undifferentiated heavy lifting of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor-lock in and unnecessary spending by researching and using open source software when appropriate.
```

Comment (misaka) [reply source/principles/architecture/index.html.md.erb:40]: Also, I believe it should be "vendor lock-in".

```suggestion
Do away with the “undifferentiated heavy lifting” of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor lock-in and unnecessary spending by researching and using open source software when appropriate.
```

File: source/principles/architecture/index.html.md.erb [hunk 3]
```diff
@@ -34,10 +34,11 @@ Design for user needs identified by research, not for generic reuse - that is pr
...
 
-## Stand on the shoulders of giants
-Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic  and make use of existing technologies where it facilitates your specific needs.  Focusing on what makes your service different is a strategic advantage, and using existing services.
+## Focus on creating value
+Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic advantages. Make use of existing technologies and services where it facilitates your specific needs.
+
+Do away with the undifferentiated heavy lifting of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor-lock in and unnecessary spending by researching and using open source software when appropriate.
```
Comment (misaka) [inline source/principles/architecture/index.html.md.erb:40]:
```suggestion
Do away with the undifferentiated heavy lifting of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor lock-in and unnecessary spending by researching and using open source software when appropriate.
```

**Review by misaka — APPROVED** [review]
🚀

**Pull request merged**